### PR TITLE
NN-1705 start time overwritten bug and scrolling issue

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ import routePaths from '../../routePaths'
 import OffenderUpload from '../OffenderUpload/OffenderUpload'
 import AddPrisoners from './AddPrisoners'
 import AppointmentDetails from '../../AppointmentDetails/AppointmentDetails'
+import ScrollToTop from '../../Components/ScrollToTop/ScrollToTop'
```
Comment (jsrobertson) [inline src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js]:
```suggestion
import ScrollToTop from '../../Components/ScrollToTop/'
```

File: src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js [hunk 2]
```diff
@@ -97,7 +98,7 @@ class AddPrisonerContainer extends Component {
     return (
       <Page title="Add prisoners" applicationTitle="Whereabouts" alwaysRender>
         {complete && (
-          <div>
+          <ScrollToTop always>
             <H3> {appointmentsCreated} appointments created </H3>
```
Comment (jsrobertson) [inline src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js]: @govuk-react/heading?

Comment (JamieDraperUK) [reply src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js]: At the moment it uses @govuk-react/heading. I just went with what Page.js uses.  I did try and upgrade to use heading as header is decrepcated now but was met with a few issues not resolving 'styled-components'. We need to upgrade to the latest gov components but maybe not for this ticket?

Comment (JamieDraperUK) [reply src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js]: Will replace heading with header as it's deprecated.

**Pull request merged**